products/firebase/api.yaml

```diff
@@ -100,3 +100,91 @@ objects:
         description: |
           The ID of the default GCP resource location for the Project. The location must be one of the available GCP
           resource locations.
+  - !ruby/object:Api::Resource
+    name: 'WebApp'
+    min_version: beta
+    base_url: projects/{{project}}/webApps
+    self_link: '{{name}}'
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      A Google Cloud Firebase web application instance
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://firebase.google.com/'
+      api: 'https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects.webApps'
```

Comment:
I see `appUrls` in the API reference, was this omission intentional?

Reply:
It was slightly intentional but also not.  I could not get my Firebase project to return any value under this property.  I can add it assuming some do, I just don't know why mine doesn't.  I've tried enabling Firebase hosting and adding custom URLs. Still, nothing is returned.